src/quo/components/text_combinations/view.cljs

```diff
@@ -7,16 +7,30 @@
     [react-native.core :as rn]))
 
 (defn icon
-  [source size]
-  [rn/image
-   {:source (if (string? source)
-              {:uri source}
-              source)
-    :style  {:border-radius 50
-             :border-width  0
-             :border-color  :transparent
-             :width         size
-             :height        size}}])
+  [source size emoji? emoji-background-color]
+  (if emoji?
+    [rn/view
+     {:style {:border-radius    50
+              :border-width     0
+              :border-color     :transparent
+              :width            size
+              :height           size
+              :justify-content  :center
+              :align-items      :center
+              :background-color emoji-background-color}}
+     [text/text
+      {:style {:margin-left 1
+               :margin-top  1}}
+      source]]
+    [rn/image
+     {:source (if (string? source)
```

Comment:
This check is unnecessary because `rn/image` already does this. cc @flexsurfer since I saw a comment of yours in a PR asking why we do this in `rn/image`.